benchmarks/BenchmarkForall.cpp

```diff
@@ -37,10 +42,18 @@ static void benchmark_openmp_loop(benchmark::State& state) {
    const int size = state.range(0);
    care::host_device_ptr<int> data(size, "data");
 
+   CARE_OPENMP_LOOP(i, 0, size) {
```

Comment:
I would add a comment on why this is being done here (I think consistency with the GPU test, where you are doing it to "warm up kernel", to avoid startup/initialization issues that made timing inconsistent).

Reply:
Fixed.